runtime/lua/vim/treesitter.lua

```diff
@@ -286,6 +286,51 @@ function M.get_captures_at_cursor(winnr)
   return captures
 end
 
+--- Returns the smallest named node at the given position
+---
+---@param opts table|nil Optional keyword arguments:
+---             - bufnr integer|nil Buffer number (nil or 0 for current buffer)
+---             - pos table|nil 0-indexed (row, col) tuple. Defaults to cursor position in the
+---                             current window. May not be omitted if {bufnr} is not the current
+---                             buffer
+---             - ignore_injections boolean Ignore injected languages (default true)
+---
+---@return TSNode|nil Node at the given position
```

Comment:
```suggestion
---@return TSNode | nil Node at the given position
```

Until we can figure out how to handle `|` in our docgen.

Reply:
Changed in 8c7f99b2c2bf777a126c61aff6a26720bec7dfd6